src/store/initialState.js

```diff
@@ -0,0 +1,7 @@
+export const signup = {
+  signupForm: {
+    fullName: '',
+    email: '',
+    password: '',
+  }
+};
```

Comment:
Why is this file in the store?

Reply:
Sorry forgot to remove it, I don't use it anymore.